gotooltest/setup_test.go

```diff
@@ -0,0 +1,30 @@
+package gotooltest
+
+import (
+	"os"
+	"testing"
+)
+
+func TestInitGoEnv(t *testing.T) {
+	// Set up a temp directory containing a bad go.mod file to
+	// ensure the working directory does not influence the probe
+	// commands run during initGoEnv
+	td, err := os.MkdirTemp("", "")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.RemoveAll(td)
+
+	// If these commands fail we are in bigger trouble
+	wd, _ := os.Getwd()
+	os.Chdir(td)
+	defer os.Chdir(wd)
```

Comment:
nit: this is a test, so use `t.Cleanup` for consistency

Reply:
Done.